logbasecommand/base.py

```diff
@@ -33,15 +33,18 @@ def log_debug(self, msg, *args, **kwargs):
         return self.logger.debug(msg, *args, **kwargs)
 
     def log(self, msg, *args, **kwargs):
-        self.__custom_stdout(msg, *args)
+        if self.logger.level <= logging.INFO:
```

Comment:
Not sure if this call shouldn't also check `if self.custom_stdout` instead.
I'm also noticing the "private" methods `__custom_stderr` aren't used here, perhaps upstream - that `BaseCommand`, that is.